build.gradle

```diff
@@ -1,190 +1,376 @@
+/*
+DO NOT CHANGE THIS FILE!
+
+Also, you may replace this file at any time if there is an update available.
+Please check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle for updates.
+ */
+
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+import java.util.concurrent.TimeUnit
+
 buildscript {
     repositories {
-        mavenCentral()
         maven {
-            name = "gt"
-            url = "https://gregtech.overminddl1.com/"
+            name = "jitpack"
```

Comment:
Do these get tried in order until it finds a match?  If so, Jitpack should be last.